### DISCUSSION TITLE
Column not found: 1054 Unknown column 'json_extract(...

### DISCUSSION THREAD

**Discussion by inalto** [spatie/laravel-translatable#425] [Q&A]
Hello,
I am using 
filament/spatie-laravel-translatable-plugin (v.3.0.87)
spatie/laravel-translatable (v.6.5.3)

on php 8.2.11 with mariadb 10.3.39

i am getting this error in the search (multilanguage) of the filament panel.

to make it work i had to modify vendor/filament/spatie-laravel-translatable-plugin/src/SpatieLaravelTranslatableContentDriver.php

like this:
//default => "json_extract({$column}, \"$.{$this->activeLocale}\")"
default => "{$column}->{$this->activeLocale}"

anyone has encountered this problem?

**Comment by dev3k** [comment]
@inalto https://github.com/filamentphp/filament/issues/9649
